x-pack/plugin/esql/qa/testFixtures/src/main/resources/conditional.csv-spec

```diff
@@ -127,3 +127,44 @@ error_rate:double | hour:date
 0.6            |2023-10-23T13:00:00.000Z
 // end::docsCaseHourlyErrorRate-result[]
 ;
+
+
+nullOnMultivaluesMathOperation#[skip:-8.12.99,reason:fixed in 8.13+]
+ROW a = 5, b = [ 1, 2 ]| EVAL sum = a + b| LIMIT 1 |  WHERE sum IS NULL;
```

Comment:
Nit: here (and below) the MV literals spacing is inconsistent with the other places (like `b = [1, 2]`)